test/gtest/uct/uct_test.cc

```diff
@@ -1099,6 +1099,12 @@ uct_listener_h uct_test::entity::listener() const {
     return m_listener;
 }
 
+uct_listener_h uct_test::entity::revoke_listener() const {
+    uct_listener_h uct_listener = listener();
+    m_listener.revoke();
+    return uct_listener;
+}
```

Comment:
no need any more

Reply:
it is needed

Reply:
it can't be done, we don't have an access to `m_listener` of `entity` which is defined as `private`.

Reply:
ic